{
  try {
    return nextOp();
  }
 catch (  IOException e) {
    return null;
  }
}
